frontend-new/src/chat/reaction/components/dislikeReasonPopover/DislikeReasonPopover.tsx

```diff
@@ -77,12 +79,12 @@ export const DislikeReasonPopover: React.FC<DislikeReasonPopoverProps> = ({
             <PrimaryButton
               key={enumValue}
               onClick={() => handleReasonClick(enumValue as DislikeReason)}
-              title={message}
+              title={t(enumValue.toLowerCase())}
```

Comment:
TBC.

Not the best way to handle this.

We are assuming that the people who will call this component, will have understanding of our languages.json